src/inject/GPMInject/interface/mini.js

```diff
@@ -16,7 +16,7 @@ window.wait(() => {
     Emitter.fire('mini', { state: true });
     oldSize = remote.getCurrentWindow().getSize();
     mainWindow.setSize(MINI_SIZE, MINI_SIZE);
-    mainWindow.setMaximumSize(MINI_SIZE, MINI_SIZE);
```

Comment:
`setAspectRatio(1)` only works on OSX as far as I am aware.  The issue isn't 1-1 ratio, I have enforced that in a resize handler.  The issue is people resizing larger than the album artwork.  The artwork will just stretch and look awful.

My question I guess is what are you trying to achieve by making the mini player not so mini :+1: